### PR TITLE
Fix test image issues

### PR DESCRIPTION
**Pull request by ngetahun**
## Description

Since December 2020, SLES15 GA images have been discontinued. The test pipeline uses these images and pipelines have been broken. This PR updates the integration test image.

Related Card: https://trello.com/c/0ZhhKlKD/1803-rmt-update-docker-image-for-integration-tests

## Change Type

*Please select the correct option.*

- [x] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [x] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [x] I have provided comments for any hard-to-understand code.
- [x] I have documented the `MANUAL.md` file with any changes to the user experience.
- [x] RMT's test coverage remains at 100%.
- [x] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by ngetahun** [comment]
[[RMT] Update Docker image for integration tests](https://trello.com/c/0ZhhKlKD/1803-rmt-update-docker-image-for-integration-tests)

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/166957/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 69ad1fd9fb9e7a7f5968c80e02d4bdde46bfe43f into 8a38a2bc5325bd64439770dc15e44343d67026eb)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/166958/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 5f9dd589d7ca47a5b7359d4ea3f2d1404091ee7d into 8a38a2bc5325bd64439770dc15e44343d67026eb)

File: integration/install.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-SUSEConnect -p sle-module-desktop-applications/15/x86_64
-SUSEConnect -p sle-module-development-tools/15/x86_64 # this and above is needed for 'rpm-build' package
+SUSEConnect -p sle-module-desktop-applications/15.2/x86_64
+SUSEConnect -p sle-module-development-tools/15.2/x86_64 # this and above is needed for 'rpm-build' package
 zypper --non-interactive ar http://download.opensuse.org/repositories/openSUSE:/Tools/SLE_15/openSUSE:Tools.repo
 zypper --non-interactive --gpg-auto-import-keys ref
 zypper --non-interactive up
 zypper --non-interactive in -t pattern devel_osc_build
-zypper --non-interactive install --no-recommend wget curl timezone \
+zypper --non-interactive install wget curl timezone \
```
Comment (wstephenson) [inline integration/install.sh:9]: This looks like it was a syntax error. Should the correct syntax `--no-recommends` be used instead?

**Review by wstephenson — APPROVED** [review]
lgtm, see comment

File: integration/install.sh (hunk 1 above)
Comment (ngetahun) [reply integration/install.sh]: Done!

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/166976/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 8c61327722f948b85e599627532b4832786ff603 into 8a38a2bc5325bd64439770dc15e44343d67026eb)

**Pull request merged**